src/hooks/auth.ts

```diff
@@ -2,9 +2,9 @@ import api from '@/utils/api';
 import { useRouter } from 'next/navigation';
 import { useEffect, useState } from 'react';
 import useSWR from 'swr';
+import { toast } from 'react-toastify';
 
 type loginProps = {
-  setErrors: any;
```

Comment:
any reason for removing this?

Reply:
Yes, because now we are not setting the error directly, whatever error is coming from the backend, we are showing the error directly from the toast message.

Reply:
Any reason for this? Should we do frontend validation also?

Reply:
Yes, we do frontend validation also if the user enter the invalid email then it's show `invalid email foramt` error using the `yup` validation schema package. And if `email` and `password` are not correct then it will show a toast message like this `These credentials do not match our records.`